vscode/src/extension.ts

```diff
@@ -67,7 +67,7 @@ import { initializeRunConfiguration, runConfigurationProvider, runConfigurationN
 import { InputStep, MultiStepInput } from './utils';
 import { PropertiesView } from './propertiesView/propertiesView';
 import { openJDKSelectionView } from './jdkDownloader';
-
+import { l10n } from './localiser';
 const API_VERSION : string = "1.0";
 const SERVER_NAME : string = "Oracle Java SE Language Server";
 export const COMMAND_PREFIX : string = "jdk";
```

Comment:
Please add the error messages in `awaitClient()` for localisation:
- `"Extension not installed."` on line 156
- `"Client not available"` on line 160